web/app/static/sass/style.sass

```diff
@@ -0,0 +1,4 @@
+@import "utilities/variables"
+
+.colorg
```

Comment:
@Greenheart nope, since it would be confusing and not clear if it's a green color or background. Altho colorgreen could be an alternative, but I found it unnecessarily long.